cookbooks/aws-parallelcluster-install/attributes/default.rb

```diff
@@ -0,0 +1 @@
+# aws-parallelcluster-install attributes
```

Comment:
By seeing this patch I'm thinking that is better to move attributes common to all the OSes here.
Let's think to third party deps, with versions etc, I think it's better to manage them from a single file.

And we can keep specific files for OS specific attributes.

Reply:
Yes, this is the goal but in this patch there are no common attribute that can be shared between different OSs. 
We have something in `cookbooks/aws-parallelcluster-common/attributes/common.rb`